<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResistorPlacement</class>
 <widget class="QDialog" name="ResistorPlacement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="4" column="1">
      <widget class="QComboBox" name="resistorPCatType_comboBox">
       <item>
        <property name="text">
         <string>PetersenCoil - 11kV - 40</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Direct - 11 kV - 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Direct - 110 kV - 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Resistor - 20 kV - 20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PetersenCoil - 20 kV - 40</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Direct - 20 kV - 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Resistor - 10 kV - 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Direct - 10 kV - 0</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>SystemID       </string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QTextEdit" name="resistorPDescription_textEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="resistorPAlias_lineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Alias</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="resistorPName_lineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Catalog type</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="resistorPUID_lineEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="resistorPSystID_lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>UID</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="resistorPButtons_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
